packages/editor/src/client/blocks/types.ts

```diff
@@ -425,7 +483,7 @@ export function blockForQueryBlock(block: FullBlockFragment | null): BlockValue
           items: block.items.map(({title, richText, image}) => ({
             id: nanoid(),
             value: {
-              title,
+              title: title,
```

Comment:
Why

Reply:
I thought I need to write it in this way (after comparing it to other code-samples). So I can just leave it with *title* ?

Reply:
Yes => https://alligator.io/js/object-property-shorthand-es6/